src/integration-tests/src/storage.rs

```diff
@@ -38,6 +40,48 @@ pub async fn buckets(builder: storage::client::ClientBuilder) -> Result<()> {
 
     cleanup_stale_buckets(&client, &project_id).await?;
 
+    let bucket_id = random_bucket_id();
+    let bucket_name = format!("projects/_/buckets/{bucket_id}");
+
+    println!("\nTesting create_bucket()");
+    let create = client
+        .create_bucket("projects/_", bucket_id)
```

Comment:
optional: As you may recall, the rate limit for this call is 0.5Hz.  We should to adjust the backoff to start a 2s, at least for this call.

Reply:
> the rate limit for this call is 0.5Hz.

I did read that. I saw we ran CRUD tests for buckets in Cloud C++ so figured it couldn't be that bad to run them here.

> adjust the backoff to start a 2s

Good idea. Done.